src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/IExceptionInstance.cs

```diff
@@ -14,5 +15,7 @@ internal interface IExceptionInstance
         string TypeName { get; }
 
         DateTime Timestamp { get; }
+
+        CallStackResult CallStackResult { get; }
```

Comment:
I think it makes more sense to have this be [CallStack](https://github.com/dotnet/dotnet-monitor/blob/main/src/Microsoft.Diagnostics.Monitoring.WebApi/Models/CallStackResults.cs#L25) instead of CallStackResult since there is only ever one call stack.